FlowersStore/Utils/Layout/UIView+Constraints.swift

```diff
@@ -0,0 +1,212 @@
+//
+//  UIView+Constraints.swift
+//  FlowersStore
+//
+//  Created by Magno Ferreira on 28/10/21.
+//
+
+import UIKit
+
+extension UIView {
+    // MARK: - Anchors and programmatically constraints
+
+    func constraintToSuperview(constant: CGFloat = 0) {
+        if let view = self.superview {
+            self
+                .topAnchor(equalTo: view.topAnchor, constant: constant)
+                .leadingAnchor(equalTo: view.leadingAnchor, constant: constant)
+                .trailingAnchor(equalTo: view.trailingAnchor, constant: -constant)
+                .bottomAnchor(equalTo: view.bottomAnchor, constant: -constant)
+        }
+    }
+
+    /// Function to add more than one view to view's hierarchy
+    func addSubviews(_ views: UIView...) {
+        views.forEach { addSubview($0) }
+    }
+
+    @discardableResult func topAnchor(equalTo anchor: NSLayoutYAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = topAnchor.constraint(equalTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func topAnchor(greaterThan anchor: NSLayoutYAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = topAnchor.constraint(greaterThanOrEqualTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func topAnchor(lessThanOrEqualTo anchor: NSLayoutYAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
```

Comment:
<!--
  0 failure: 
  1 warning:  Function 'topAnch...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Utils/Layout/UIView+Constraints.swift;
  Line: 42;
-->

- :warning: Function 'topAnchor(lessThanOrEqualTo:constant:priority:)' is unused